### PR TITLE
CT 116: Update VET TEC Profile Page Content

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a comparison tool user, I need the VET TEC profile page to display relevant information in a user friendly manner so that I can make decisions about which provider / program I would like to attend to use my benefits.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19868

## Testing done
Testing conducted using local host

## Screenshots
Screen Shot 2019-11-01 at 9 02 13 AM
Screen Shot 2019-11-01 at 9 07 32 AM

## Acceptance criteria Approved Programs
- [x] In the table listing the programs, the length is displayed in weeks.
- [x] The benefits estimator indents the percentage listing descriptions.
- [x] The housing allowance includes "(monthly)" as part of the label instead of "/mo" on the amount.
## Acceptance criteria Application Process
- [x] The sentence "After you've been approved for VET TEC, apply to the program you'd like to attend." is replaced with "Then, after you've been approved for VET TEC, apply to the VA-approved training provider you'd like to attend."
- [x] The sentence link "Learn more about VET TEC programs" is replaced with the following: "To learn more about available programs, visit the training provider's website."
- [x] "visit the training provider's website" is a link to the provided website.
## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx [hunk 1]
```diff
@@ -1,62 +1,100 @@
...
+            After you’ve been approved for VET TEC, apply to the program you’d
+            like to attend.
+          </p>
+        ) : (
+          <p>
+            Then, after you've been approved for VET TEC, apply to the
+            VA-approved training provider you'd like to attend.
+          </p>
+        )}
+        {environment.isProduction() ? (
```
Comment (shawkey-daniel) [inline src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: Need prod flag comment here also

File: src/applications/gi/components/vet-tec/VetTecCalculator.jsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import { getCalculatedBenefits } from '../../selectors/vetTecCalculator';
 import VetTecCalculatorForm from './VetTecCalculatorForm';
 import PropTypes from 'prop-types';
 import { ariaLabels } from '../../constants';
+import environment from '../../../../platform/utilities/environment';
```
Comment (shawkey-daniel) [inline src/applications/gi/components/vet-tec/VetTecCalculator.jsx]: Need to use the platform alias path here

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx [hunk 3]
```diff
@@ -1,62 +1,100 @@
...
-      in the mail if we approve your application.
-    </p>
-    <p>
-      <a
-        href={
-          '/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/introduction'
+class VetTecApplicationProcess extends React.Component {
+  providersWebsiteLink = () => {
+    const programs = this.props.institution.programs;
+    return programs[0].providerWebsite === null ? (
```
Comment (shawkey-daniel) [inline src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: Could the providerWebsite be empty?

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx [hunk 4]
```diff
@@ -1,62 +1,100 @@
...
+      <p>
+        To learn more about available programs, visit the training provider's
+        website.
+      </p>
+    ) : (
+      <p>
+        To learn more about available programs,{' '}
+        <a
+          href={` https://${programs[0].providerWebsite}`}
```
Comment (shawkey-daniel) [inline src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: Extra space after the tic

File: src/applications/gi/components/vet-tec/VetTecCalculator.jsx [hunk 5]
```diff
@@ -140,24 +145,48 @@ export class VetTecCalculator extends React.Component {
...
         </button>
       </div>
+      {// PROD FLAG CT 116 STORY 19868
+      environment.isProduction() ? (
+        <div>
+          <div className="row calculator-result">
+            <div className="small-8 columns">
+              <div>In person rate:</div>
+            </div>
+            <div className="small-4 columns vads-u-text-align--right small-screen:vads-u-padding-left--7">
```
Comment (shawkey-daniel) [inline src/applications/gi/components/vet-tec/VetTecCalculator.jsx]: This long className is used four times. Might want to define a constant.

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx (hunk 4 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: I need that space or the linked text and the previous word will be combined.

File: src/applications/gi/components/vet-tec/VetTecCalculator.jsx (hunk 2 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/vet-tec/VetTecCalculator.jsx]: updated

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: updated

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx (hunk 3 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: Added empty string check

File: src/applications/gi/components/vet-tec/VetTecCalculator.jsx (hunk 5 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/vet-tec/VetTecCalculator.jsx]: Broke it out into separate variable

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx (hunk 4 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: Nevermind, I was able to make it work without that space.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Review by U-DON — APPROVED** [review]

**Pull request merged**